.github/workflows/linter-yaml.yml

```diff
@@ -1,125 +0,0 @@
-name: "[linter] YAML files"
-
-on:
-  pull_request:
-
-env:
-  PYTHONPATH: ${{github.workspace}}
-  PYVER: "3.8"
-  CONFIG_FILES_PATH: "recipes/*/config.yml"
-  CONANDATA_FILES_PATH: "recipes/*/*/conandata.yml"
-
-jobs:
-  test_linter:
-    # A job to run when the linter changes. We want to know in advance how many files will be broken
-    name: Test linter changes (YAML files)
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Get changed files
-        uses: ./.github/actions/pr_changed_files
-        id: changed_files
-        with:
-          files: |
-            linter/**
-
-      - uses: actions/setup-python@v4
-        if: steps.changed_files.outputs.any_changed == 'true'
-        with:
-          python-version: ${{ env.PYVER }}
-
-      - name: Install dependencies
-        if: steps.changed_files.outputs.any_changed == 'true'
-        run: pip install yamllint strictyaml argparse
-
-      - name: Run linter (config.yml)
-        if: steps.changed_files.outputs.any_changed == 'true' && always()
-        run: |
-          echo "::add-matcher::linter/yamllint_matcher.json"
-          yamllint --config-file linter/yamllint_rules.yml -f standard ${{ env.CONFIG_FILES_PATH }}
```

Comment:
AFAIK, yamllint is not run by conan-center hooks, and it catches real errors, which have to prevent PR from being merged, for example in case of key duplication.

Reply:
we need to split error for categories (it should be possible, right?)
- style errors like extra space should be plain warnings, they shouldn't fail a job
- errors like duplicated keys should be real errors, blocking further build (and perhaps, aborting build execution)